src/test/specs/API-v3/05 - EE-6347 Total Monthly Payments for Salaried Assessments.feature

```diff
@@ -0,0 +1,74 @@
+            | 2017-11-30 | 775.00 |             | 07           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-10-27 | 775.00 |             | 06           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-09-29 | 775.00 |             | 05           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-08-25 | 775.00 |             | 04           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-07-28 | 775.00 |             | 03           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-06-30 | 775.00 |             | 02           | FP/Ref1        | Flying Pizza Ltd |
+            | 2017-05-26 | 775.00 |             | 01           | FP/Ref1        | Flying Pizza Ltd |
+        And the applicants partner has the following income records:
+            | Date       | Amount | Week Number | Month Number | PAYE Reference | Employer        |
+            | 2018-04-30 | 775.00 |             | 12           | RM/Ref3        | Reliable Motors |
+            | 2018-03-27 | 775.00 |             | 11           | RM/Ref3        | Reliable Motors |
+            | 2018-02-25 | 775.00 |             | 10           | RM/Ref3        | Reliable Motors |
+            | 2018-01-30 | 775.00 |             | 09           | RM/Ref3        | Reliable Motors |
+            | 2017-12-27 | 775.00 |             | 08           | RM/Ref3        | Reliable Motors |
+            | 2017-11-29 | 775.00 |             | 07           | RM/Ref3        | Reliable Motors |
+            | 2017-10-27 | 775.00 |             | 06           | RM/Ref3        | Reliable Motors |
+            | 2017-09-25 | 775.00 |             | 05           | QE/Ref4        | Quality Estates |
+            | 2017-08-28 | 775.00 |             | 04           | QE/Ref4        | Quality Estates |
+            | 2017-07-27 | 775.00 |             | 03           | RM/Ref3        | Reliable Motors |
+            | 2017-06-30 | 775.00 |             | 02           | RM/Ref3        | Reliable Motors |
+            | 2017-05-29 | 775.00 |             | 01           | RM/Ref3        | Reliable Motors |
+        When the Income Proving v3 TM Family API is invoked with the following:
+            | NINO - Applicant        | RR223611A  |
+            | NINO - Partner          | GG199882B  |
+            | Application Raised Date | 2018-04-30 |
+        Then The Income Proving TM Family API provides the following result:
+            | HTTP Response                     | HTTP Status               | 200                               |
+            | Applicant                         | National Insurance Number | RR223611A                         |
+            | Partner                           | National Insurance Number | GG199882B                         |
+            | Category A Monthly Salary         | Financial requirement met | false                             |
+            | Category A Monthly Salary         | Failure Reason            | NON_CONSECUTIVE_MONTHS            |
```

Comment:
Shouldn't this failure be due to having Multiple Employers instead?

Reply:
👍